src/main/java/uk/gov/digital/ho/hocs/info/utils/EmailDomainValidator.java

```diff
@@ -12,7 +13,7 @@
 public class EmailDomainValidator implements ConstraintValidator<EmailDomainWhitelisted, String> {
 
     @Value("${user.email.whitelist}")
-    private List<String> whitelistedDomains = asList("homeoffice.gov.uk");
+    private List<String> whitelistedDomains = List.of("homeoffice.gov.uk");
```

Comment:
Will this default to homeoffice.gov.uk if the config var is not set? I wonder if either "disabling user creation if domains not given" or "allowing any email" would be more sensible defaults?

Reply:
Changed to default to an empty list. This disables when domains are not provided.